packages/customer-erasure/src/cli.js

```diff
@@ -0,0 +1,148 @@
+import fs from 'fs'
+import { getCredentials } from '@commercetools/get-credentials'
+import pino from 'pino'
+import PrettyError from 'pretty-error'
+import yargs from 'yargs'
+
+import CustomerErasure from './main'
+import { description } from '../package.json'
+
+process.title = 'customer-erasure'
+
+const args = yargs
+  .usage(
+    `
+Usage: $0 [options]
+${description}`
+  )
+  .showHelpOnFail(false)
+  .help('help', 'Show help text.')
+  .version()
+  .option('output', {
+    alias: 'o',
+    default: 'stdout',
+    describe: 'Path to output file.',
+  })
+  .option('customerId', {
+    alias: 'c',
+    describe: 'Customer to fetch or delete.',
+    demand: true,
+  })
+  .option('apiUrl', {
+    default: 'https://api.sphere.io',
+    describe: 'The host URL of the HTTP API service.',
+  })
+  .option('authUrl', {
+    default: 'https://auth.sphere.io',
+    describe: 'The host URL of the OAuth API service.',
+  })
+  .option('accessToken', {
+    describe: 'CTP client access token.',
+  })
+  .option('projectKey', {
+    alias: 'p',
+    describe: 'API project key.',
+    demand: true,
+  })
+  .option('deleteAll', {
```

Comment:
Given this option, we should prompt the user for confirmation, as it’s a non-reversible action! ⚠️

Reply:
Ahh, great idea!

Reply:
yargs does not have native prompt so I added the functionality by using [prompt-confirm](https://www.npmjs.com/package/prompt-confirm)

Reply:
I usually use `prompts` but, yeah I guess it's fine :)